### PR TITLE
Editing pass

### PR DESCRIPTION
**Pull request by cgpotts**
This PR is for an editing pass. 

My main goal was to try to beef up the connections between our model and the humans from our corpus. I was able to add basic prose for this in section 4; it would be great to substantiate these things with some basic statistical models related to the patterns in Table 2.

I added various to-dos (with `\todocheck`) and tried to flesh out some of the other arguments and examples.

### CODE REVIEW

File: writing/2016/tacl_colors.tex [hunk 1]
```diff
@@ -362,7 +379,7 @@ \section{Task and data collection}\label{sec:corpus}
     \label{fig:basic-rsa:l1}
   \end{subfigure}
   %
-  \caption{An example of RSA applied to color reference.}
+  \caption{An example of RSA applied to an idealized color reference task.}
```
Comment (futurulus) [inline writing/2016/tacl_colors.tex]: Do we need to say the task is "idealized"? This is an actual example from our corpus (with utterance = "blue"). Perhaps the RSA process has been simplified--no alpha or costs, a small alternative utterance set.

File: writing/2016/tacl_colors.tex [hunk 2]
```diff
@@ -557,13 +580,14 @@ \subsection{Base speaker}\label{sec:s0}
 and an affine transformation and softmax function are applied to the output to
 produce a probability distribution predicting the following token of the description.
 The model is substantively similar to well-known models for image caption generation
-\cite{Karpathy2015,Vinyals2015}, but using a second recurrent neural network for producing
+\cite{Karpathy2015,Vinyals2015}, in using a second recurrent neural network for producing
```
Comment (futurulus) [inline writing/2016/tacl_colors.tex]: This clause specifies the differences from the image captioning models, not the similarities--the image captioning models cited all use convolutional neural nets for the input, and feed in the input representation only at the beginning. We could probably just delete the "but..." if it feels like a liability to you.

**Comment by cgpotts** [comment]
We should signal somehow that the display is not showing model output and that the utterance set is artificial and artificially small.

File: writing/2016/tacl_colors.tex (hunk 2 above)
Comment (cgpotts) [reply writing/2016/tacl_colors.tex]: Can you rewrite it to be clearer? I thought those other models also in effect chained two RNNs.

Comment (futurulus) [reply writing/2016/tacl_colors.tex]: Can do.

**Comment by futurulus** [comment]
Got it. Added a clarification mentioning this.

**Pull request merged**